03_qgep_db_dss.sql

```diff
@@ -1887,6 +1887,8 @@ ALTER TABLE qgep_od.reach ADD COLUMN clear_height  integer ;
 COMMENT ON COLUMN qgep_od.reach.clear_height IS 'Maximal height (inside) of profile / Maximale Innenhöhe des Kanalprofiles / Hauteur intérieure maximale du profil';
 ALTER TABLE qgep_od.reach ADD COLUMN coefficient_of_friction  smallint ;
 COMMENT ON COLUMN qgep_od.reach.coefficient_of_friction IS 'yyy http://www.linguee.com/english-german/search?source=auto&query=reibungsbeiwert / Hydraulische Kenngrösse zur Beschreibung der Beschaffenheit der Kanalwandung. Beiwert für die Formeln nach Manning-Strickler (K oder kstr) / Constante de rugosité selon Manning-Strickler (K ou kstr)';
+ALTER TABLE qgep_od.reach ADD COLUMN swmm_default_coefficient_of_friction  smallint ;
```

Comment:
Could we move these statements for additonal attributes in [05_datamodel_extension](https://github.com/QGEP/datamodel/blob/master/05_data_model_extensions.sql) to not mix creation of VSA-DSS datamodel and other attributes in this file?

Reply:
Done